src/pages/index.tsx

```diff
@@ -175,14 +174,17 @@ const HomePage = (): jsx.JSX.Element => (
           </LinkButton>
         </>
       </BoxContainer>
-      <BirthdayThrasher />
       <BoxContainer
         theme="dark"
         background={{ backgroundColor: `${neutral[97]}` }}
       >
         <>
           <InnerText title="We're reader funded" theme="dark">
-            <p>
+            <p
```

Comment:
Sure want to cut this width short rather than make paragraph the width of divider lines like other blocks of text?

Reply:
I don't quite understand this one

Reply:
Sorry wasn't clear, just resolve, it was just a tiny observation versus anything else. I just meant the css (below it) has 850px width set specifically but all other paragraph blocks on this page just fill the width, presume thats exactly as intended. I should have placed  this comment on the css line below to make it clearer.